db/migrate/20170208080437_add_last_check_notifications_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastCheckNotificationsAtToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :last_check_notifications_at, :datetime, default: DateTime.now
```

Comment:
What do you mean?
I added it because there will be a lot of bugs if this column will not have default value.
What do you mean when you said 'same'. At the begin this date will be the same as the created_at, but it will update when user checks his feed